docs/examples/webextension/background.js

```diff
@@ -0,0 +1,18 @@
+var EXPERIMENT_ID = 'webextension-example-1';
+
+function sendTelemetryPing(data) {
+  document.getElementById('tp-proxy').contentWindow.postMessage({
+    op: 'queueTelemetryPing',
+    data: {
+      subject: EXPERIMENT_ID,
+      data: data
+    }
+  }, '*');
```

Comment:
In the "real world" implementation, I assume we would tell devs to only post the message to `targetOrigin` of `testpilot.firefox.com` ?